app/views/providers/allocations/index.html.erb

```diff
@@ -64,7 +64,7 @@
 
     <p class="govuk-body">
       <% if @training_providers.present? %>
-        Request fee-funded PE for 2021/22 for any organisations not currently offering this course.
+        Request fee-funded PE for 2021&thinsp;–&thinsp;2022 for any organisations not currently offering this course.
```

Comment:
Is it me or this reads bad?
also `organisations` vs `organisation`

Reply:
If there is only one organisation listed do we need to say 'organisation' rather then 'organisations'

Reply:
Paul and Jesse review this content and its correct. I think it `organisations` because they have an option from multiple organisations not just one organisation.